src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx

```diff
@@ -10,18 +10,42 @@ import contextMapThreeEntities from 'images/context-map-three-entities.png'
 
 import contextMapEntityDetail from 'images/context-map-entity-detail.png'
 
-Some New Relic features, like distributed tracing and incident intelligence, display topological maps that show relationships between your New Relic-monitored entities and other external hosts and services. These maps can help you diagnose and correct issues.
+import omnimapServiceMaps from 'images/omnimapServiceMaps.png'
+
+import omnimapAutomapTooltip from 'images/omnimapAutomapTooltip.png'
+
+import omnimapServiceMapsTopNav from 'images/omnimapServiceMapsTopNav.png'
+
+import omnimapServiceMapsRightPane from 'images/omnimapServiceMapsRightPane.gif'
+
+New Relic can take data from instrumented entities and create topological maps that display the relationships between them. These entity-scoped maps use a source entity as an entry point into the rest of your system, letting you understand how your hosts, services, and applications relate and affect each other.
+
+If you want to learn more about Workloads maps, then check out [Intro to Workloads](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one). If you're looking to learn about automap, then read [Automap for troubleshooting entities](/docs/new-relic-solutions/new-relic-one/ui-data/automaps).
```

Comment:
Tbh this was a low view doc that I commandeered for omnimap, but I agree that it'll be very helpful for customers to have a list of these experiences. 

I'm going to add a section at the bottom that links to these other maps, since they're actually documented once in their own docs.